decko/features/support/env.rb

```diff
@@ -20,7 +20,7 @@
 # newer version of cucumber-rails. Consider adding your own code to a new file
 # instead of editing this one. Cucumber will automatically load all
 # features/**/*.rb files.
-Before("@background-jobs or @delayed-jobs or @javascript") do |scenario|
+Before("@delayed-jobs or @javascript") do |scenario|
```

Comment:
Have you thought this through, changing the database cleaner strategy for @background-jobs? If you wanted to remove it then you should remove it completely or, what I prefer in this case, comment the rest of background-jobs out an leave a comment about this change in case it will be needed again. I don't like an unused api to be crippled.